README.md

```diff
@@ -37,26 +22,29 @@ https://dev.futurice.com:8081/vimma/
 
 ## Running on your machine:
 
-Create a Python3 virtual enviroment and use it for most commands.
```

Comment:
I wouldn't lose the commands below showing exactly what and how to run. The other stuff (vagrant, run.py) are conveniences for these commands.
Should we move them to their own section “doing everything by hand” at the end?